Examples/CombinationModulesPipeline.md

```diff
@@ -0,0 +1,46 @@
+# Combination modules Pipeline 
+## An example of pipelines mixing a pipeline with a combination of module commands with JSON & Search
+
+### Connect to the Redis server:
+```csharp
+var redis = ConnectionMultiplexer.Connect("localhost");
+```
+
+### Setup pipeline connection
+```csharp
+var pipeline = new Pipeline(ConnectionMultiplexer.Connect("localhost"));
+```
+
+### Add JsonSet to pipeline
+```csharp
+pipeline.Json.SetAsync("person:01", "$", new { name = "John", age = 30, city = "New York" });
+pipeline.Json.SetAsync("person:02", "$", new { name = "Joy", age = 25, city = "Los Angeles" });
+pipeline.Json.SetAsync("person:03", "$", new { name = "Mark", age = 21, city = "Chicago" });
+pipeline.Json.SetAsync("person:04", "$", new { name = "Steve", age = 24, city = "Phoenix" });
+pipeline.Json.SetAsync("person:05", "$", new { name = "Michael", age = 55, city = "San Antonio" });
+```
+
+### Create the schema to index first and age as a numeric field
+```csharp
+var schema = new Schema().AddTextField("name").AddNumericField("age", true).AddTagField("city");
+```
+
+### Filter the index to only include Jsons and prefix of person
+```csharp
+var parameters = FTCreateParams.CreateParams().On(Literals.Enums.IndexDataType.JSON).Prefix("person:");
+```
+
+### Create the index via pipeline
+```csharp
+pipeline.Ft.CreateAsync("person-idx", parameters, schema);
+```
+
+### Search for all indexed person records
+```csharp
+var getAllPersons = pipeline.Ft.SearchAsync("person-idx", new Query());
```

Comment:
I think we should add a few lines of using the value we got in `getAllPersons` To show the user how to use the value we received with the help of the pipeline.